dpdispatcher/lsf.py

```diff
@@ -62,14 +64,25 @@ def gen_script_header(self, job):
 
         return lsf_script_header
 
-    def do_submit(self, job):
+    def do_submit(self, job, retry=0, max_retry=3):
         script_file_name = job.script_file_name
         script_str = self.gen_script(job)
         job_id_name = job.job_hash + '_job_id'
         self.context.write_file(fname=script_file_name, write_str=script_str)
-        stdin, stdout, stderr = self.context.block_checkcall(
-            'cd %s && %s %s' % (self.context.remote_root, 'bsub < ', script_file_name)
-        )
+        
+        try:
+            stdin, stdout, stderr = self.context.block_call(
```

Comment:
You changed `block_checkcall` to `block_call`, but `block_call` returns four values instead of three. Did you test your code?

Reply:
Seems to be a typo, for I just did not want to change the code here. Let me fix it.